<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-FV335H8K" failures="1" tests="1" name="Test1" time="12.731" errors="0" timestamp="21 Dec 2021 14:36:14 GMT">
  <testcase classname="FourTest" name="testfour" time="8.322">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at FourTest.testfour(FourTest.java:20)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- testfour -->
</testsuite> <!-- Test1 -->
